StreamChatClient_v3/Config/ChatClientConfig.swift

```diff
@@ -47,3 +47,25 @@ extension ChatClientConfig {
         // something
     }
 }
+
+/// The API key of the app. You can obtain this value ... TODO
+public struct APIKey: Equatable {
+    /// The string representation of the API key
+    public let apiKeyString: String
+    
+    /// Creates a new `APIKey` from the provided string. Fails, if the string is empty.
+    public init?(_ apiKeyString: String) {
```

Comment:
Why do we need this to return optional? Why not log.assert?

Reply:
How would this work in release configuration? You need to return something. The whole system down the road relies on the fact the `APIKey` can't be empty.

Reply:
Just like how it works with `StaticString` init

Reply:
fixed in 56b3602c29959d9de334960beefb70742e61dc2d